### PR TITLE
Jkmarx/analysis tab url fix

### LINKED ISSUE #2027: Analysis Filtering url is not picked up by File Browser 2 facet filtering
Commit: `c33d8d5f9dd88168c140c5e99114c5e61e535d72`

@mccalluc Analysis facet filtering in FB2 is not triggered with the newly created urls. It seems like the logic to create the url on facet selection is there, but the selections aren't updated with the inverse manner.

### PR DESCRIPTION
**Pull request by jkmarx**
Resolves #2027

### CODE REVIEW

**Comment by jkmarx** [comment]
@scottx611x analysis-redirect-fix

File: refinery/core/models.py [hunk 1]
```diff
@@ -1387,6 +1387,15 @@ def data_sets_query(self):
         )
         return quote(json.dumps({analysis_facet_name: self.uuid}))
 
+    def facet_name(self):
```
Comment (scottx611x) [inline refinery/core/models.py]: Considering how the method is named, it might be worth being explicit and decorating this method as a `@property` ?

Comment (scottx611x) [reply refinery/core/models.py]: Also should reuse `facet_name` [here](https://github.com/refinery-platform/refinery-platform/blob/develop/refinery/core/models.py#L1383) now that we have it

File: refinery/core/models.py [hunk 2]
```diff
@@ -1379,12 +1379,15 @@ def get_input_file_uuid_list(self):
...
-    def data_sets_query(self):
-        analysis_facet_name = '{}_{}_{}_s'.format(
+    def facet_name(self):
+        return '{}_{}_{}_s'.format(
             NodeIndex.ANALYSIS_UUID_PREFIX,
             self.data_set.get_latest_study().id,
             self.data_set.get_latest_assay().id,
         )
+
+    def data_sets_query(self):
```
Comment (mccalluc) [inline refinery/core/models.py]: Do we even want to keep this around at all? Is it ever the right way of doing what it's trying to do?

**Comment by jkmarx** [comment]
@mccalluc Good point, the dash board should be updated accordingly.

**Pull request merged**